packages/manager/modules/pci/src/projects/project/instances/instances.controller.js

```diff
@@ -7,6 +7,7 @@ export default class CloudProjectComputeInfrastructureListCtrl {
   /* @ngInject */
   constructor(
     $q,
+    $state,
```

Comment:
**question:** Is it required? Looks like it's unused.

Reply:
It is not required. So I removed it. [938025d](https://github.com/ovh/manager/commit/938025d1c9be8b80406a969faae827de5ca064c1)